### PR TITLE
ref(org-stats): New layout for page

### PR DESCRIPTION
**Pull request by leedongwei**
## Normal
image

## Narrower
image

## Narrowest
image

### CODE REVIEW

File: static/app/views/usageStats/usageStatsOrg.tsx [hunk 1]
```diff
@@ -303,47 +310,41 @@ class UsageStatsOrganization extends AsyncComponent<Props, State> {
       },
       {
         title: t('Accepted'),
-        description: tct(
-          'Accepted [dataCategory] were successfully processed by Sentry.',
-          {dataCategory}
-        ),
+        help: tct('Accepted [dataCategory] were successfully processed by Sentry.', {
```
Comment (JoshFerge) [inline static/app/views/usageStats/usageStatsOrg.tsx:313]: nit: this tooltip has a period at the end while others dont 🙃

**Review by JoshFerge — COMMENTED** [review]
accepted attachments quantity needs a bit more space i think but other than that lgtm
Screen Shot 2021-04-21 at 8 27 02 PM

File: tests/js/spec/views/usageStats/usageChart/utils.spec.jsx [hunk 2]
```diff
@@ -70,4 +70,9 @@ describe('getXAxisDates', () => {
       'Jul 9 8:00 PM - 9:00 PM',
     ]);
   });
+
+  it('recovers from odd arguments', () => {
```
Comment (leedongwei) [inline tests/js/spec/views/usageStats/usageChart/utils.spec.jsx:74]:
```suggestion
  it('handles invalid date strings', () => {
```

**Review by doralchan — APPROVED** [review]

File: static/app/views/usageStats/usageStatsOrg.tsx (hunk 1 above)
Comment (leedongwei) [reply static/app/views/usageStats/usageStatsOrg.tsx:313]:
```suggestion
        help: tct('Accepted [dataCategory] were successfully processed by Sentry', {
```

**Comment by maheskett** [comment]
could `X in last min` be a row above the accepted amount? otherwise, the mobile layout is a little cut off due to the project breakdown table, but that could be a follow-up.

Screen Shot 2021-04-22 at 10 49 30 AM

**Review by maheskett — APPROVED** [review]

**Comment by leedongwei** [comment]
Fix for `X in last min`: https://github.com/getsentry/sentry/pull/25593

Project breakdown is a 1-line fix on `grid-template-columns`, but holding off as I'm still playing around with it to avoid a horizontal scrolling table.

**Pull request merged**